docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -1,17 +1,104 @@
 # Enabling Cody on Sourcegraph Enterprise
 
-<p className="subtitle">Learn how to use Cody and its features with Sourcegraph enterpise.</p>
+<p className="subtitle">This guide will walk you through the steps to install and set up Cody with your Sourcegraph Enterprise instance.</p>
 
-Cody enhances your coding experience by providing intelligent code suggestions, context-aware completions, and advanced code analysis. This guide will walk you through the steps to install and set up Cody with your Sourcegraph enterprise instance via:
-
-- [Self-hosted Sourcegraph](#cody-on-self-hosted-sourcegraph-enterprise)
-- [Sourcegraph Cloud](#cody-on-sourcegraph-cloud)
+Cody enhances your coding experience by providing intelligent code suggestions, context-aware completions, and advanced code analysis.
 
 <LinkCards>
    <LinkCard href="https://about.sourcegraph.com/cody/pricing" imgSrc="https://sourcegraph.com/.assets/img/sourcegraph-mark.svg" imgAlt="Cody Enterprise" title="Cody Enterprise" description="Get in touch with our team to try Cody for Sourcegraph Enterprise." />
 </LinkCards>
 
-## Cody on self-hosted Sourcegraph enterprise
+## Cody Enterprise features
+
+To cater to your Enterprise requirements, Cody Enterprise offers the following features:
+
+### IDE Token Expiry
+
+<Callout type= "info">IDE Token Expiry is supported with VS Code and JetBrains extensions.</Callout>
```

Comment:
This applies to access tokens in general not just when in use with Cody. I previously updated the site config docs to show these options.

Reply:
Thanks I'll remove the callout.